views/displaycard/main.js

```diff
@@ -0,0 +1,54 @@
+let profile_owner_name = document.getElementById("profile-owner-name");
+let profile_contents = document.getElementById("profile-contents");
+
+// Values to be connected with the DB
+profile_owner_name.innerText = "Leah Kazenmayer";
+let profile_labels = ["email", "phone number", "facebook", "instagram", "twitter", "linkedin", "github", "personal website"];
+let profile_values = ["kazenml1@tcnj.edu", "(123) 456 - 7890", "lelekaz", "kazy2001", "leletweet", "https://www.linkedin.com/in/leah-kazenmayer/", "https://github.com/lelekaz", "lelekaz.com"];
+
+
+for (let i = 0; i < profile_labels.length; i++) {
+    let profile_info_div = document.createElement("div");
+    profile_info_div.classList.add("profile-info-div");
+
+    let icon_i = document.createElement("i");
+    switch (profile_labels[i].toLowerCase()) {
```

Comment:
Yup, this works. I'm sure there's probably a more efficient way of checking the values but this is perfect.